Makefile

```diff
@@ -73,3 +73,24 @@ clean:
 .PHONY docker-build:
 docker-build:
 	@docker build -t genn:latest .
+
+BASE := 12.2.0-devel-ubuntu22.04
+PY_VER := 3.11
+.PHONY ubuntu_wheel:
+ubuntu_wheel:
+	@docker build -f Dockerfile.ubuntu_builder \
+		--build-arg BASE=$(BASE) \
+		--build-arg PY_VER=$(PY_VER) \
+		--target=output --output type=local,dest=dist/ .
+
+CUDA := 12.2
+.PHONY wheels:
+wheels:
+	@docker build -f Dockerfile.builder \
+		--build-arg CUDA=$(CUDA) \
+		--target=output --output type=local,dest=dist/ .
+
+# TODO: Consider build with docker run instead of docker build
```

Comment:
What would the advantages of that be?

Reply:
I'm not sure yet, beyond saving a bit of time in image building. It was the method used in the example but I'd need to investigate a bit more.